src/plugins/home/server/services/sample_data/routes/install.ts

```diff
@@ -148,8 +153,9 @@ export function createInstallRoute(
 
         const client = getClient({ includedHiddenTypes });
 
+        const savedObjects = sampleDataset.savedObjects as SavedObjectsBulkCreateObject[];
         createResults = await client.bulkCreate(
-          sampleDataset.savedObjects.map(({ version, ...savedObject }) => savedObject),
+          savedObjects.map(({ version, ...savedObject }) => savedObject),
```

Comment:
`@kbn/config-schema` creates a result type for `{ property: schema.any() }`, as `{property?: any}`. That's okay, since `any` includes `undefined` or `null` types, but it's not compatible with `savedObject.attributes` property,(which is a generic, with `unknown` default).